### PR TITLE
Reduce the size of Iaso docker images

### PR DESCRIPTION
**Pull request by olethanh**
We were adding the node_modules of Polio and other useless stuff in the image.

### CODE REVIEW

**Comment by olethanh** [comment]
should be unneed with #195

**Comment by olethanh** [comment]
nevermind we still need it for the Webpack docker

File: .dockerignore [hunk 1]
```diff
@@ -66,6 +66,14 @@ target/
 __pycache__/
 local_settings.py
 db.sqlite3
-media
 
+# data that get mounted
+media/
+testdata/
```
Comment (madewulf) [inline .dockerignore:72]: We actually use testdata in the readme, i think it should stay

Comment (olethanh) [reply .dockerignore]: ok put it back

**Pull request merged**